Web/Views/Usuario/_Cadastrar.cshtml

```diff
@@ -0,0 +1,41 @@
+@{
+    Layout = null;
+}
+
+<div class="cadastrar-container">
+  <form id="cadastroUsuario">
+      <h1>Crie sua conta</h1>
+      <hr>
+
+      <label for="nome"><b>Nome</b></label>
+      <input type="text" placeholder="Seu nome" name="nome" id="Nome" required>
```

Comment:
Mesma coisa sobre o aspnet.
Utilizando o asp-for no lugar de name, pode excluir o id="Nome"

Reply:
Mesma coisa para o restante do formulario.